ocw/lib/openqa.py

```diff
@@ -26,7 +18,7 @@ def get_url(server):
             got = openqa_client.client.requests.head(
                 url,
                 timeout=5,
-                verify=bool(os.environ.get('REQUESTS_CA_BUNDLE')),
+                verify=False,
```

Comment:
You meant this, right?
```suggestion
                verify=True,
```

Reply:
I was also puzzled by this but this is valid.

The reason is that `pcw` needs to query any openQA instances for job status updates. This includes openQA development instances, where many people use self-signed certificates.

Reply:
OK for me as well then. We might find a better overall solution to everything later.
However, I would appreciate to have a short comment why TLS verification is disabled whenever this is done in any code. @ricardobranco777 I would appreciate very much if you could add a short comment to the sections.